plugins/Monolog/Formatter/LineMessageFormatter.php

```diff
@@ -81,12 +81,15 @@ private static function formatTrace(array $trace, $numLevels = 10)
             }
 
             $level = $trace[$i];
+            $levelTrace = '';
             if (isset($level['file'], $level['line'])) {
                 $levelTrace = '#' . $i . (str_replace(PIWIK_DOCUMENT_ROOT, '', $level['file'])) . '(' . $level['line'] . ')';
-            } else {
+            } elseif (isset($level['class']) && isset($level['type']) && isset($level['function'])) {
```

Comment:
could be simplified to
```suggestion
            } elseif (isset($level['class'], $level['type'], $level['function'])) {
```

Reply:
sure that will work too (might not have much benefit though)